device.go

```diff
@@ -414,12 +414,9 @@ func (d *device) AppInstall(ipaPath string) (err error) {
 
 	installationPath := path.Join(stagingPath, fmt.Sprintf("%s.ipa", bundleID))
 
-	var data []byte
-	if data, err = os.ReadFile(ipaPath); err != nil {
-		return err
-	}
-	if err = d.afc.WriteFile(installationPath, data, AfcFileModeWr); err != nil {
-		return err
+	if r, err := os.Open(ipaPath); err != nil {return err} else {
+		defer r.Close()
+		if err = d.afc.WriteFile(installationPath, r, AfcFileModeWr); err != nil {return err}
 	}
```

Comment:
🤔 It would be better to close the file as soon as possible

```golang
	_ipaFile, err := os.Open(ipaPath)
	if err != nil {
		return err
	}
	if err = d.afc.WriteFile(installationPath, _ipaFile, AfcFileModeWr); err != nil {
		_ipaFile.Close()
		return err
	}
	_ipaFile.Close()
```